hadoop-ozone/dist/src/main/compose/test-all.sh

```diff
@@ -46,9 +46,12 @@ for test in $(find "$SCRIPT_DIR" -name test.sh | grep "${OZONE_TEST_SELECTOR:-""
       echo "ERROR: Test execution of $(dirname "$test") is FAILED!!!!"
   fi
   RESULT_DIR="$(dirname "$test")/result"
-  cp "$RESULT_DIR"/robot-*.xml "$RESULT_DIR"/docker-*.log "$RESULT_DIR"/*.out* "$ALL_RESULT_DIR"/
+  rebot -N $(basename $(dirname "$test")) -o "$ALL_RESULT_DIR"/$(basename $(dirname "$test")).xml "$RESULT_DIR"/*.xml
```

Comment:
Nit: store result of `$(basename $(dirname "$test"))` in variable.

Reply:
done (got `${d}` with the rebase...)